playwright/tests/customers.spec.ts

```diff
@@ -0,0 +1,234 @@
+  await customersPage.gotoCustomerDetailsPage(CUSTOMERS.deleteCustomer.id);
+  await customersPage.deleteCustomer();
+  await customersPage.deleteDialog.clickDeleteButton();
+  await customersPage.expectSuccessBanner();
+  await customersPage.goToCustomersListView();
+  await customersPage.searchForCustomer(CUSTOMERS.deleteCustomer.email);
+  await expect(customersPage.emptyDataGridListView).toBeVisible();
+});
+
+test("TC: SALEOR_205 Bulk delete customers @e2e @customer", async () => {
+  const customersToBeBulkDeleted = CUSTOMERS.customersToBeBulkDeleted.names;
+
+  await customersPage.goToCustomersListView();
+  await customersPage.typeInSearchOnListView("bulk-delete");
+
+  const rowsToCheck = [0, 1, 2];
+
+  await customersPage.checkGridCellTextAndClick(0, rowsToCheck, customersToBeBulkDeleted);
+  await customersPage.clickBulkDeleteGridRowsButton();
+  await customersPage.deleteDialog.clickDeleteButton();
+  await customersPage.expectSuccessBanner();
+  await expect(customersPage.emptyDataGridListView).toBeVisible();
+});
+
+test("TC: SALEOR_206 As an admin I want to add address to the customer and set it as default shipping @e2e @customer", async () => {
+  await customersPage.gotoCustomerDetailsPage(CUSTOMERS.editCustomer.id);
+  await addressesListPage.clickManageAddresses();
+  await addressesListPage.clickAddAddressButton();
+
+  const addressUK = ADDRESS.addressUK;
+
+  await addressForm.completeBasicInfoAddressForm(addressUK);
+  await addressForm.typeCompanyName(addressUK.companyName);
+  await addressForm.typePhone(addressUK.phone);
+  await addressForm.typeAddressLine2(addressUK.addressLine2);
+  await addAddressDialog.clickConfirmButton();
+
+  const addedAddress = addressesListPage.savedAddress.filter({
+    hasText: addressUK.lastName,
+  });
+  const addedAddressCard = addressesListPage.addressCard.filter({
+    hasText: addressUK.lastName,
+  });
+
+  await expect(addedAddress).toBeVisible();
+  await addressesListPage.clickShowMoreMenu(addressUK.lastName);
+  await addressesListPage.setAsDeafultShippingAddress();
+  await expect(addedAddressCard.locator(addressesListPage.addressTypeTitle)).toHaveText(
+    "Default Shipping Address",
+  );
+});
+
+test("TC: SALEOR_209 As an admin I want to update customer's address and set it as default billing @e2e @customer", async () => {
+  await customersPage.gotoCustomerDetailsPage(CUSTOMERS.editCustomer.id);
+  await addressesListPage.clickManageAddresses();
+  await addressesListPage.clickShowMoreMenu(CUSTOMERS.editCustomer.initialShippingAddress.lastName);
+  await addressesListPage.clickEditAddress();
+
+  const newAddress = ADDRESS.addressUS;
+
+  await addressForm.completeBasicInfoAddressForm(newAddress);
+  await addressForm.typeCompanyName(newAddress.companyName);
+  await addressForm.typePhone(newAddress.phone);
+  await addressForm.typeAddressLine2(newAddress.addressLine2);
+  await addressForm.selectCountryArea(newAddress.countryArea);
```

Comment:
please check all duplication connected with the addressForm

Reply:
It will be handled in the next PR